scripts/changelog/rename_changelog_file.py

```diff
@@ -37,7 +37,6 @@ def rename_files(path):
 
 if renamed_features_count + renamed_bugfixes_count + auto_renamed_bugfixes_count + auto_renamed_features_count > 0:
     repository = git.Repo('.')
-    repository.git.add('changelog/unreleased')
-    repository.git.add('libnavui-androidauto/changelog/unreleased')
+    repository.git.add()
```

Comment:
What if we accidentally add files that shouldn't be added?
Can we just add a directory only if it exists? Or maybe even create one if it doesn't (although I'm not sure if empty directories are allowed).

Reply:
Oh, you can create an empty file. It will be deleted in the release process, won't have any affect on changelog (at lease it shouldn't) and will create all the directories (if we place an empty file both to bugfixes, features, knows inssues and other). Do you think that would work?

Reply:
We can't commit empty directory

Reply:
Fixed